lib/team.rb

```diff
@@ -0,0 +1,12 @@
+require 'csv'
```

Comment:
We can move this `require` to the spec file since `Team` isn't reading any CSV data itself